constants/auth-constants.ts

```diff
@@ -0,0 +1,5 @@
+export enum SessionStatus {
+  Authenticated = 'authenticated',
+  Loading = 'loading',
+  unauthenticated = 'unauthenticated'
```

Comment:
I just noticed that this name is lowercase but the other ones are pascal case.

Reply:
Nice catch. I'll fix it ASAP.